tests/test_worksheet.py

```diff
@@ -116,3 +117,19 @@ def test_create_test_results_links(self):
             "https://qe-component-readiness.dptools.openshift.org/sippy-ng/component_readiness/main?view=4.15-qe-auto-release",
             tr._ws.get(LABEL_SIPPY_AUTO_RELEASE, value_render_option="FORMULA")[0][0],
         ) 
+
+    def test_update_cell_with_hyperlinks(self):
```

Comment:
For testing I would suggest to use testing spreadsheet instead of release template.
In addition the test doesn't get the spreadsheet to the initial state.

Reply:
I would also suggest move to move the testing method to `TestTestReport` class as both `update_shipment_info` as well as `update_cell_with_hyperlinks` methods are in `TestReport` class, not in `WorksheetManager` class. See https://github.com/openshift/release-tests/pull/583